_includes/misc/purchase-premium-plugins.md

```diff
@@ -0,0 +1,51 @@
+{% if extensionType != 'Packs' %}
+{% assign extensionType = 'plugin' %}
+{% endif %}
+{% if pluralExtensionType == true %}
+{% assign isAre = 'are' %}
+{% else %}
+{% assign isAre = 'is' %}
+{% endif %}
+### Getting started with TinyMCE {{pluginname}}
+
+#### Creating an account
+
+To try out the **{{pluginname}} {{extensionType}}** and the {{site.cloudname}}, create a free [{{site.accountpage}}]({{site.accountsignup}}). Sign-up for a free {{site.accountpage}} and receive a 30-day trial of the {{site.companyname}} [premium plugins]({{site.plugindirectory}}), skins, and icon packs; with no credit card information or commitment required.
+
+#### Buy TinyMCE {{pluginname}}
```

Comment:
This seems to be incorrect for `Tiny Drive` as it says `Buy TinyMCE Tiny Drive`.

Reply:
fixed with if statements